docs/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Closes #(number) - (issue name)
```

Comment:
Bit of a hassle to get the issue name and GitHub will actually let you hover over to see the issue anyway

Reply:
Also can this file be moved to `/.github` folder to go with the other templates